src/components/events/EventCard/style.module.scss

```diff
@@ -0,0 +1,52 @@
+.container {
+  background: transparent;
+  border: 1px solid var(--theme-primary-6);
+  border-radius: 10px;
+  -ms-overflow-style: none;
+  overflow-x: hidden;
+  padding: 0;
+  position: relative;
+  scrollbar-width: none;
+  text-align: left;
+  width: 20rem;
+
+  .image {
+    flex-shrink: 0;
+    height: 9.375rem;
+    overflow: none;
+    position: relative;
+    width: 20rem;
+  }
+
+  .info {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    padding: 1rem;
+
+    .header {
+      align-items: center;
+      display: flex;
+      flex-direction: row;
+      gap: 0.5rem;
+
+      .eventDetails {
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
The ellipses don't show for me

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/17f65fba-4ec4-4a12-a731-2eb66d04e980)

It seems you have to set these two CSS properties directly on the `Typography` elements themselves in order for it to work

Reply:
fixed o7